src/corva/handlers.py

```diff
@@ -576,6 +576,11 @@ def _merge_events(
     Only "scheduled" and "stream" type of apps can be processed here.
     If somehow any other type is passed - raise an exception
     """
+    if not isinstance(
```

Comment:
Why should we run functionality to merge event if event does not have any data to merge?

Reply:
@corva-alex-s, partial merge rerun events handler was implemented before events merging functionality. What stopped you from learning how partial merge rerun functionality works before working on events merging functionality?